test/data/BUILD

```diff
@@ -20,7 +20,7 @@ load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 filegroup(
```

Comment:
Is the test data going to be used for anything other than integration tests in the foreseeable future?

If not, this whole package should live in the `integration` folder. (Note that currently the files are all visible only to the subpackages of `test/integration`.)

Reply:
Good point! That says it all - done.